pkg/remote/aws/route53_health_check_enumerator.go

```diff
@@ -0,0 +1,46 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type Route53HealthCheckEnumerator struct {
+	repository repository.Route53Repository
+	factory    resource.ResourceFactory
+}
+
+func NewRoute53HealthCheckEnumerator(repo repository.Route53Repository, factory resource.ResourceFactory) *Route53HealthCheckEnumerator {
+	return &Route53HealthCheckEnumerator{
+		repo,
+		factory,
+	}
+}
+
+func (e *Route53HealthCheckEnumerator) SupportedType() resource.ResourceType {
+	return aws.AwsRoute53HealthCheckResourceType
+}
+
+func (e *Route53HealthCheckEnumerator) Enumerate() ([]resource.Resource, error) {
+	healthChecks, err := e.repository.ListAllHealthChecks()
+	if err != nil {
+		return nil, remoteerror.NewResourceEnumerationErrorWithType(err, string(e.SupportedType()), aws.AwsRoute53HealthCheckResourceType)
+	}
+
+	results := make([]resource.Resource, len(healthChecks))
+
+	for _, healthCheck := range healthChecks {
+		results = append(
+			results,
+			e.factory.CreateAbstractResource(
+				string(e.SupportedType()),
+				*healthCheck.Id,
+				map[string]interface{}{},
```

Comment:
You do not need to pass anything here I think

Reply:
I think it can break think if we produce nil Attribute (and data is just casted in the factory)

Reply:
Yeah it's defined like this in every other enumerators